docs/getting-started.md

```diff
@@ -33,7 +33,7 @@ Place the lingo binary on your $PATH.
 
 In order to run Tenets against your repository, your lingo client will need to authenticate with the CodeLingo servers. To do so, you are required to have an account. Please follow these steps to set up your client:
 
-1. Create a CodeLingo account <a href="https://codelingo.io/join" target="_blank">here</a>
+1. Create a CodeLingo account by signing in with GitHub <a href="https://codelingo.au.auth0.com/login?state=g6Fo2SBNWGhoMUtWMTFFWDJ1SEVSWVdNendRUmE3VUs3bFVlc6N0aWTZIDhab3k0WkZlNUQ2T2c4ZUN3YjFlekQ3cE56STNPZGNEo2NpZNkgZUNDeUdQWjlZYjIwdEVPaVlraTZ0b0MyME5BVnBSd0c&client=eCCyGPZ9Yb20tEOiYki6toC20NAVpRwG&protocol=oauth2&response_type=token%20id_token&redirect_uri=https%3A%2F%2Fwww.codelingo.io%2Fcallback&scope=openid%20username%20email%20write%3Acli-token&audience=https%3A%2F%2Fflow.codelingo.io%2Fcodelingo%2Flogin%2Fgogs&nonce=kCoDkl38eVHLDT0rBGb3naFCIeJiPTZC&auth0Client=eyJuYW1lIjoiYXV0aDAuanMiLCJ2ZXJzaW9uIjoiOS44LjIifQ%3D%3D" target="_blank">here</a>
```

Comment:
RM the link. And write:

1. Create a CodeLingo account: navigate to <a href="https://codelingo.io" target="_blank">codelingo.io</a> and click on the "Sign in with GitHub" button.